project/Common.scala

```diff
@@ -28,7 +28,7 @@ object Common {
       if(isSnapshot.value) {
         Seq(
           "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-          "ASF Snapshots" at "https://repository.apache.org/content/groups/snapshots"
+          "ASF spark" at "https://repository.apache.org/content/repositories/orgapachespark-1341" // Apache spark repo
```

Comment:
`https://repository.apache.org/content/repositories/orgapachespark-1341` gives me a 404.  What is this change trying to do?

Reply:
I'm inheriting this change from the original spark-3.0.0 branch, @mengxr, do you remember what this is about?